lib/parser/resolver.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Thrift.Parser.Resolver do
+  alias Thrift.Parser.ParsedFile
+  alias Thrift.Parser.Models.{
+    Field,
+    StructRef
+  }
+
+  def start_link do
+    Agent.start_link(&Map.new/0, name: __MODULE__)
+  end
+
+  def stop do
+    Agent.stop(__MODULE__)
+  end
+
+  def print do
+    Agent.get(__MODULE__, &IO.inspect(&1))
+  end
+
+  def add(f=%ParsedFile{}) do
```

Comment:
I see you somethings write `x=%{...}` and sometimes `%{...}=x`. We should use a consistent style.